pkg/api/vlabs/upgradetypesandvalidate.go

```diff
@@ -0,0 +1,27 @@
+package vlabs
+
+import "fmt"
+
+// UpgradeContainerService API model
+type UpgradeContainerService struct {
+	OrchestratorProfile *OrchestratorProfile `json:"orchestratorProfile,omitempty"`
+}
+
+// Validate implements APIObject
+func (ucs *UpgradeContainerService) Validate() error {
```

Comment:
Does this need to take the existing containerService model to evaluate against? I'm cool with punting, just wanted to mention.